playground/mocks/config/responseConfig.js

```diff
@@ -1069,6 +1069,13 @@ const selectOktaVerifyMethod = {
   ],
 };
 
+const smartCardEnrollOrVerify = {
+  '/idp/idx/introspect': [
+    'authenticator-enroll-smartcard',
+    // 'authenticator-verification-smartcard',
```

Comment:
Did you want to uncomment this for verification?

Reply:
When we want to test Verify in playground, we can uncomment `authenticator-verification-smartcard` and comment `authenticator-enroll-smartcard`